src/Uno.UI/Controls/CommandBarNavigationItemRenderer.iOS.cs

```diff
@@ -177,6 +177,12 @@ public TitleView()
 			}
 		}
 
+		protected override void OnBeforeArrange()
+		{
+			//This is to ensure that the layouter gets the correct **finalRect**
```

Comment:
@davidjohnoliver  I believe it's because of this: https://github.com/unoplatform/uno/blob/master/src/Uno.UI/Controls/CommandBarNavigationItemRenderer.iOS.cs#L128

We are setting the CommandBar as the **Parent** for the `TitleView`  but when rendered this is not a direct child of the `CommandBar`.